Resources/Prototypes/SimpleStation14/Loadouts/jobs.yml

```diff
@@ -150,6 +153,7 @@
   exclusive: true
   jobWhitelist:
     - MedicalDoctor
+    - ChiefMedicalOfficer
```

Comment:
The CMO coat is quite literally their most recognizable item beyond their hardsuit, and I feel allowing them to simply replace it with a regular lab coat would be foolish. Besides, theirs is _far_ shwankier.

```suggestion
```